woocommerce/woocommerce
# Set correct timezone when backfilling data.

### All Submissions:






### Changes proposed in this Pull Request:



### How to test the changes in this Pull Request:

1. With a timezone other than UTC+0, enable COT and enable sync.
2. Create a new order or update an existing one.
3. Inspect the modified date for the posts table, post_modified_gmt will contain an offset timestamp. With this PR, post_modified_gmt will the same as date_updated_gmt in the orders table.

Note that instead of writing a new unit test, I have modified OrderTableDataStore tests to run with an offset timezone which reproduces the error.

### Other information:




### FOR PR REVIEWER ONLY:
